src/js/game.ts

```diff
@@ -54,7 +54,8 @@ export interface SleuthVoices {
 const SLEUTHS: Array<Sleuths> = [
   {
     'nonGoogleVoice': 'Samantha',
-    'googleVoice': 'Google US English',
+    'googleVoice': document.documentElement.lang === 'ja' ?
```

Comment:
For all the cases where we are doing a lang check do you think it might be a little cleaner to have a util function for `isLangJa` in utils.ts and call that vs explicitly doing the check here in all the files. It's minor but might make the readability a bit better. WDYT?

Reply:
@jbruwer Agreed. It would be better for readability and flexibility in changing the logic for lang detection in the future.

Reply:
Done.